static/js/pages/candidates-office.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+/* global context */
+
+var $ = require('jquery');
+
+var tables = require('../modules/tables');
+var columns = require('../modules/columns');
+var helpers = require('../modules/helpers');
+
+var FilterPanel = require('fec-style/js/filter-panel').FilterPanel;
+var filterTags = require('fec-style/js/filter-tags');
+
+var candidatesTemplate = require('../../templates/candidates-office.hbs');
+
+var columnOptions = {
+  name: {
+    data: 'name',
+    className: 'all',
+    width: '280px',
+    render: function(data, type, row, meta) {
+      return tables.buildEntityLink(
+        data,
+        helpers.buildAppUrl(
+          ['candidate', row.candidate_id],
+          tables.getCycle(row.election_years, meta)
+        ),
+        'candidate'
+      );
+    }
+  },
+  party: {data: 'party_full', className: 'min-tablet hide-panel'},
+  state: {data: 'state', className: 'min-desktop hide-panel'},
+  district: {data: 'district', className: 'min-desktop hide-panel'},
+  receipts: tables.currencyColumn({data: 'receipts', className: 'min-tablet'}),
+  disbursements: tables.currencyColumn({data: 'disbursements', className: 'min-tablet'}),
+  trigger: {
+    className: 'all u-no-padding',
+    width: '20px',
+    orderable: false,
+    render: function(data, type, row, meta) {
```

Comment:
'meta' is defined but never used.<br>'row' is defined but never used.<br>'type' is defined but never used.<br>'data' is defined but never used.